packages/block-library/src/comments-pagination-previous/block.json

```
@@ -38,6 +38,23 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"spacing": {
+			"margin": true,
+			"padding": true
+		},
```

```suggestion
		"spacing": {
			"margin": true,
			"padding": true,
			"__experimentalDefaultControls": {
				"margin": false,
				"padding": false
			}
		},
```

Following other similar blocks, it might be best to hide these controls by default.

Spacing controls are normally displayed by default so need to be explicitly hidden here.